crates/python/qcs_sdk/compiler/quilc.pyi

```diff
@@ -233,6 +211,55 @@ async def compile_program_async(
     """
     ...
 
+@final
+class CompilationResult:
+    @property
+    def program(self) -> str:
+        """ """
+        ...
+    @property
+    def native_quil_metadata(self) -> Optional[NativeQuilMetadata]:
+        """ """
```

Comment:
might as well omit the empty comments?

Reply:
Oops, yeah, those should be filled in, fixed.